language/compiler/ir_to_bytecode/src/compiler.rs

```diff
@@ -797,6 +867,7 @@ fn compile_command(
             compile_expression(context, function_frame, code, *e)?;
 
             let def_idx = context.struct_definition_index(&name)?;
+            record_srcloc_info!(context, code, cmd.span);
```

Comment:
we need to record a loc everytime we push right? 

Maybe make a macro for that?

Reply:
We actually don't!

As an example take an `if` expression. We want the `Branch` for this to be associated with the consequent branch of source-level expression. Thus, there is a many-to-one mapping of bytecode offset to source location. So having this isn't as useful IMO.